charts/storage-apps/Chart.yaml

```
@@ -17,7 +17,7 @@ dependencies:
 annotations:
   artifacthub.io/changes: |
```

can you add the security annontation:
```suggestion
  artifacthub.io/containsSecurityUpdates: "true"
  artifacthub.io/changes: |
```

and maybe also some details and links for the versions we are skipping, they also contain features and potentially relevant fixes

---

done